src/Rules/TestRuleHelper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ikvasnica\PHPStan\Rules;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPUnit\Framework\TestCase;
+
+final class TestRuleHelper
+{
+    /** @var string */
+    private const TEST_CLASS_ENDING_STRING = 'Test';
+
+    public static function isUnitTest(string $namespace, string $className, string $unitTestNamespacePattern): bool
+    {
+        return Strings::contains($namespace, $unitTestNamespacePattern)
+            && Strings::endsWith($className, self::TEST_CLASS_ENDING_STRING);
+    }
+
+    public static function isCallToAssert(Node $node, Scope $scope): bool
+    {
+        if (! $node instanceof Node\Expr\MethodCall && ! $node instanceof Node\Expr\StaticCall) {
```

Comment:
Ja bych odstranil mezeru za !, ty mi trha srdce :-D

Reply:
Hele to je nějakej CS rule :D Mně se to celkem začlo líbit, bije to do očí, takže vidíš, že je tam zápor :)